README.md

```diff
@@ -41,33 +45,52 @@ Install CMake and build the C++ tools:
     sudo apt-get install python3-yaml
     sudo pip3 install -r requirements.txt
 
+This step is known to fail with a compiler error while building the `pyjson5`
+library when using Arch Linux and Fedora. `pyjson5` needs one change to build
+correctly:
+
+    git clone https://github.com/Kijewski/pyjson5.git
+    cd pyjson5
+    sed -i 's/char \*PyUnicode/const char \*PyUnicode/' src/_imports.pyx
+    sudo make
+
+This might give you and error about `sphinx_autodoc_typehints` but it should
+correctly build and install pyjson5. After this, run either option 1 or 2 again.
+
+### Step 6: ###
 Always make sure to set the environment for the device you are working on before
 running any other commands:
 
     source database/artix7/settings.sh
 
+### Step 7: ###
 **If you are a prjxray developer, skip this step.** But, if you just want to use the Python API with a pre-generated database, just download a current stable version and skip the rest of the instructions: 
 
     # WARNING: causes git issues, overrides environment
     ./download-latest-db.sh
 
+### Step 8: ###
 (Re-)creating the entire database (this will take a very long time!):
 
     cd fuzzers
     make -j$(nproc)
 
-(Re-)creating parts of the database, for example LUT init bits:
+If you want to (re-)create parts of the database, you first need to run:
```

Comment:
This is incomplete. Tilegrid is definitely an important dependency. FWIW we just fixed some of the issues that made download-latest-db not good for general use, so maybe we can simplify the instructions to recommend everyone to do that. Maybe remove this small bit right now and address that in a separate PR?

Reply:
Ok, so then the steps would be to always download the latest database (or build everything yourself) and then you can choose to build parts yourself?

Reply:
Yeah. I think given recent fixes, general quick start should now always download latest db